.github/workflows/rust.yml

```diff
@@ -158,6 +158,22 @@ jobs:
           command: test
           args: --no-default-features --no-fail-fast --no-run
 
+  cw-compile:
```

Comment:
Why this has been added? Isn't the existing `cw-check` job sufficient?

Reply:
Reverted at 9b83d22 in favor to keep `cw-check` contained.